i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-firewalls-juniper-ssg-snmp.md

```diff
@@ -2,36 +2,628 @@
+| Traffic-Generic-Name | Net-JuniperSSG-Traffic-Generic-Name-custom | Contrôle de la bande passante de l'interface. Pour chaque contrôle apparaîtra le nom de l'interface              |            |
+| Traffic-Global       | Net-JuniperSSG-Traffic-Global-custom       | Contrôle de la bande passante d'un ensemble d'interfaces. Pour chaque contrôle apparaîtra le nom de l'interface  |            |
+| Vpn-Status           | Net-JuniperSSG-Vpn-Status-custom           | Contrôle le statut des tunnels VPN                                                                               | X          |
+| Vpn-Usage            | Net-JuniperSSG-Vpn-Usage-custom            | Contrôle le trafic entrant et sortant des tunnels VPN                                                            |            |
+
+> Les services listés ci-dessus ne sont pas créés automatiquement lorsqu'un modèle d'hôte est appliqué. Pour les utiliser, [créez un service manuellement](/docs/monitoring/basic-objects/services) et appliquez le modèle de service souhaité.
+
+> Si la case **Découverte** est cochée, cela signifie qu'une règle de découverte de service existe pour ce service.
+
+</TabItem>
+</Tabs>
+
+### Règles de découverte
+
+#### Découverte d'hôtes
+
+| Nom de la règle | Description                                                                                                                                                                                                                                  |
+|:----------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| SNMP Agents     | Discover your resource through a SNMP subnet scan. You need to install the [Generic SNMP](./applications-protocol-snmp.md) connector to get the discovery rule and create a template mapper for the **Net-Juniper-SSG-custom** host template |
+
+Rendez-vous sur la [documentation dédiée](/docs/monitoring/discovery/hosts-discovery) pour en savoir plus sur la découverte automatique d'hôtes.
+
+#### Découverte de service
+
+| Nom de la règle          | Description |
+|:-------------------------|:------------|
+| Net-Juniper-SSG-Vpn-Name |             |
+
+Rendez-vous sur la [documentation dédiée](/docs/monitoring/discovery/services-discovery)
+pour en savoir plus sur la découverte automatique de services et sa [planification](/docs/monitoring/discovery/services-discovery/#règles-de-découverte).
+
+### Métriques & statuts collectés
+
+Voici le tableau des services pour ce connecteur, détaillant les métriques rattachées à chaque service.
+
+<Tabs groupId="sync">
+<TabItem value="Cpu" label="Cpu">
+
+| Métrique    | Unité |
+|:------------|:------|
+| cpu_1min    | %     |
+| cpu_5min    | %     |
+| cpu_15min   | %     |
+
+</TabItem>
+<TabItem value="Hardware" label="Hardware">
+
+Coming soon
```

Comment:
No metrics? 
If none, Explain that there are no metrics for this service template.